packages/main/test/specs/MultiComboBox.spec.js

```diff
@@ -1483,6 +1483,31 @@ describe("MultiComboBox general interaction", () => {
 			await mcb.click();
 			assert.notOk(await popover.getProperty("opened"), "Popover with valueStateMessage should not be opened.");
 		});
+
+		it("Should apply correct text to the tokens overflow indicator", async () => {
+			const mcNItems = await $("#mc-items");
+			const mcNMore = await $("#mc-more");
+			const tokenizerNItems = await mcNItems.shadow$("ui5-tokenizer");
+			const tokenizerNMore = await mcNMore.shadow$("ui5-tokenizer");
+			const nItemsLabel = await tokenizerNItems.shadow$(".ui5-tokenizer-more-text");
+			const nMoreLabel = await tokenizerNMore.shadow$(".ui5-tokenizer-more-text");
+			let resourceBundleTextItems = null;
+			let resourceBundleTextMore = null;
+			
+	
+			resourceBundleTextItems = await browser.executeAsync(done => {
```

Comment:
you can concat this in one browser.executeAsync, just pass a {} as a result of done.